js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -289,6 +247,23 @@ const EditingController = modules.ViewController.inherit((function() {
             this.component._optionsByReference[EDITING_CHANGES_OPTION_NAME] = true;
         },
 
+        getDefaultEditorTemplate: function() {
```

Comment:
getDefaultEditorTemplate -> _getDefaultEditorTemplate because it is used in this module only.